src/js/view/controls/settings-menu.js

```diff
@@ -160,9 +160,13 @@ export function setupSubmenuListeners(settingsMenu, controlbar, model, api) {
     model.change('playbackRates', onPlaybackRatesChange, this);
     model.change('playbackRate', (changedModel, playbackRate) => {
         const rates = model.get('playbackRates');
-        if (rates) {
+        const provider = model.getVideo();
+        if (rates && provider.supportsPlaybackRate) {
             activateSubmenuItem('playbackRates', rates.indexOf(playbackRate));
         }
+        if (!provider.supportsPlaybackRate) {
```

Comment:
We don't need to do this here. We determine when to show the menu in the `onPlaybackRatesChange` callback.